.github/workflows/eventarc-audit-storage.yaml

```diff
@@ -0,0 +1,45 @@
+name: eventarc-audit-storage
+on:
+  push:
+    branches:
+    - main
+    paths:
+    - 'eventarc/audit-storage/**'
+  pull_request:
+    paths:
+    - 'eventarc/audit-storage/**'
+  pull_request_target:
+    types: [labeled]
+  schedule:
+  - cron:  '0 2 * * *'
+jobs:
```

Comment:
I generally recommend you add the following to prevent queue backup:

```yaml
concurrency:
  group: '${{github.workflow}}-${{ github.head_ref || github.ref }}'
  cancel-in-progress: true
```

This applies to all workflows, but I'll only leave the comment once here.